source/geometry_model/box.cc

```diff
@@ -221,6 +221,16 @@ namespace aspect
 
 
     template <int dim>
+    double
+    Box<dim>::height_above_original_surface(const Point<dim> &position) const
+    {
+      const double d = (position(dim-1)-box_origin[dim-1]) - extents[dim-1] ;
+      return std::min (std::max (d, -maximal_depth()), maximal_depth());
```

Comment:
what is your reason for limiting the return of height above the surface to the maximal depth? 

Although I can imagine that the mesh will be probably horribly deformed, the purpose of this function seem to be just for visualisation. I therefore think that  it should just return what the height above the surface is, so that it can output the true height. Currently it will show caped off values in that file, which might not be expected by the user. 

The same argumentation can be used for also not limiting it to the -minus depth. The reason why we limit the depth is that we know depth should not be smaller then zero and larger then the maximum depth. I do not think that is the case for the height above a original/reference surface.

Reply:
Sure, I'll take out the max height limit. As for the low limit, I think it can't be less than -extent[dim-1] because we're calculating topography from "sea level", not from original topography.

Reply:
Good point. If you know for sure it shouldn't every be less then -extent[dim-1], I think you  should use an assert (see #705 on how this was done before for the depth function).